Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -345,118 +345,118 @@ PixelShaderUid GetPixelShaderUid()
   return out;
 }
 
-void ClearUnusedPixelShaderUidBits(APIType ApiType, const ShaderHostConfig& host_config,
+void ClearUnusedPixelShaderUidBits(APIType api_type, const ShaderHostConfig& host_config,
                                    PixelShaderUid* uid)
 {
   pixel_shader_uid_data* const uid_data = uid->GetUidData();
 
   // OpenGL and Vulkan convert implicitly normalized color outputs to their uint representation.
   // Therefore, it is not necessary to use a uint output on these backends. We also disable the
   // uint output when logic op is not supported (i.e. driver/device does not support D3D11.1).
-  if (ApiType != APIType::D3D || !host_config.backend_logic_op)
+  if (api_type != APIType::D3D || !host_config.backend_logic_op)
     uid_data->uint_output = 0;
 
   // If bounding box is enabled when a UID cache is created, then later disabled, we shouldn't
   // emit the bounding box portion of the shader.
   uid_data->bounding_box &= host_config.bounding_box & host_config.backend_bbox;
 }
 
-void WritePixelShaderCommonHeader(ShaderCode& out, APIType ApiType, u32 num_texgens,
+void WritePixelShaderCommonHeader(ShaderCode& out, APIType api_type, u32 num_texgens,
                                   const ShaderHostConfig& host_config, bool bounding_box)
 {
   // dot product for integer vectors
-  out.Write("int idot(int3 x, int3 y)\n"
-            "{\n"
-            "\tint3 tmp = x * y;\n"
-            "\treturn tmp.x + tmp.y + tmp.z;\n"
-            "}\n");
-
-  out.Write("int idot(int4 x, int4 y)\n"
-            "{\n"
-            "\tint4 tmp = x * y;\n"
-            "\treturn tmp.x + tmp.y + tmp.z + tmp.w;\n"
-            "}\n\n");
+  out.WriteFmt("int idot(int3 x, int3 y)\n"
```

Comment:
I initially considered this when putting the fmt interface in place. I eventually concluded that this would be an inconsistent API experience and makes it easier for people to mentally forget to double-up the braces in scenarios that they're needed. I opted for consistency instead where they get doubled up in all cases.

It would also be even more inconvenient to change that now in the middle of a migration, given code exists where this would retroactively apply to it.

Reply:
Yeah, braces being consistently doubled up makes reviewing these migration PRs easier too. IMO, it is easier to check that every `{` in the original code has been replaced with a `{{` if that applies to every brace, not just *some* of them.